mapie/tests/test_regression.py

```diff
@@ -502,30 +503,42 @@ def test_aggregate_with_mask_with_prefit() -> None:
     """
     Test ``_aggregate_with_mask`` in case ``cv`` is ``"prefit"``.
     """
-    mapie_reg = MapieRegressor(cv="prefit")
+    mapie_reg = MapieRegressor(LinearRegression().fit(X, y), cv="prefit")
+    mapie_reg = mapie_reg.fit(X, y)
```

Comment:
QUESTION: why the previous version does not work with your changes?

Reply:
`_aggregate_with_mask` is a method of `EnsembleRegressor` therefore it is not possible to call it without fitting the estimator (the instance `estimator` does not exist if `MapieRegressor` is not fitted).

Reply:
I understand, thank you for this clarification :)